ts/components/conversation/SessionRecording.tsx

```diff
@@ -76,6 +76,36 @@ const StyledFlexWrapper = styled.div<StyledFlexWrapperProps>`
   }
 `;
 
+function RecordingDurations({
+  isRecording,
+  displaySeconds,
+  remainingSeconds,
+}: {
+  isRecording: boolean;
+  displaySeconds: number;
+  remainingSeconds: number;
+}) {
+  const displayTimeString = useFormattedDuration(displaySeconds, { forceHours: false });
+  const remainingTimeString = useFormattedDuration(remainingSeconds, { forceHours: false });
+
+  return (
+    <div className={classNames('session-recording--timer', !isRecording && 'playback-timer')}>
+      {displayTimeString + (remainingTimeString ? ` / ${remainingTimeString}` : '')}
+    </div>
+  );
+}
+
+function RecordingTimer({ displaySeconds }: { displaySeconds: number }) {
+  const displayTimeString = useFormattedDuration(displaySeconds, { forceHours: false });
+
+  return (
+    <div className={classNames('session-recording--timer')}>
+      {displayTimeString}
```

Comment:
Localizer

Reply:
we can't not a localized string
![image](https://github.com/user-attachments/assets/c04bb251-96c5-4c20-a3c1-3ce567f71ec6)

Reply:
Ah my mistake. Resolving